storagenode/piecestore/endpoint.go

```diff
@@ -685,7 +685,9 @@ func min(a, b int64) int64 {
 
 // ignoreEOF ignores io.EOF error.
 func ignoreEOF(err error) error {
-	if err == io.EOF {
+	// gRPC gives us an io.EOF but dRPC gives us a wrapped io.EOF
+	// so we must try to unwrap it first to be backwards compatible.
+	if err := errs.Unwrap(err); err == io.EOF {
```

Comment:
`errs.Unwrap` unwraps at most one layer. Use `errs.Is` instead.

Reply:
Done!